tests/integration/states/test_cmd.py

```diff
@@ -46,9 +46,11 @@ def test_run_hide_output(self):
         '''
         cmd.run with output hidden
         '''
+
+        cmd = u'dir' if IS_WINDOWS else u'ls'
```

Comment:
Since we have the following import at the top:
```
from __future__ import absolute_import, print_function, unicode_literals
```
you don't have to add the `u`. 
`unicode_literals` makes all strings unicode strings.

Reply:
I know you're just following convention here.